.golangci.yaml

```diff
@@ -11,8 +14,42 @@ linters-settings:
     suggest-new: true
   misspell:
     locale: US
+    ignore-words:
+      - cancelled
+      - metre
+      - meter
+      - metres
+      - kilometre
+      - kilometres
```

Comment:
why is it ignored?

Reply:
I've added those configs to have a linear linter configuration between Otel projects, as mentioned at the referenced issue.
https://github.com/open-telemetry/opentelemetry-go-build-tools/blob/29a46c89ab69347be7d6f2c7f4f94135cb4d0d4f/.golangci.yml